### PR TITLE
Blog post for standalone release process

### PR DESCRIPTION
**Pull request by gaiksaya**
### Description
Blog post for standalone release process
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2023-09-12-standalone-components-release-process.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+layout: post
+title:  "Automating the release process for standalone components in OpenSearch: Addressing challenges and implementing solutions"
+authors: 
+  - sayaligaikawad
+  - billbeckler
+date: 2023-09-12 01:01:01 -0700
+categories: 
+    - releases
+    - community
+twittercard:
```
Comment (pajuric) [inline _posts/2023-09-12-standalone-components-release-process.md:11]: Please add the following meta: 
**meta_keywords:** OpenSearch release process, centralized CI/CD system, standalone components in OpenSearch

**meta_description:** Discover the steps the OpenSearch Project took to implement a centralized CI/CD system to create a repeatable and scalable release process for standalone components.

Comment (gaiksaya) [reply _posts/2023-09-12-standalone-components-release-process.md:11]: Done!

Comment (pajuric) [inline _posts/2023-09-12-standalone-components-release-process.md:11]: @nateynateynate - Do we need the Twitter card and description on Line 12? Please advise.

**Review by krisfreedain — APPROVED** [review]
Looks good. Made a few minor edits - dropped Overview to fix how this shows up on the main blog page, added a '/' so the image displays, and added brackets so the additional resources links are clickable.

**Pull request merged**